test/compto-test-client/test_client.js

```diff
@@ -214,4 +223,39 @@ async function dailyDistributionEvent() {
     let dailyDistributionEventResult = await sendAndConfirmTransaction(connection, dailyDistributionEventTransaction, [testuser_keypair, testuser_keypair]);
     console.log("DailyDistributionEvent transaction confirmed", dailyDistributionEventResult);
 
+}
+
+async function getValidBlockHashes() {
+    let data = Buffer.alloc(1);
+    data.writeUInt8(Instruction.GET_VALID_BLOCKHASHES, 0);
+    console.log("data: ", data);
+    let keys = [
+        // the Global Comptoken Data Account (also mint authority)
+        { pubkey: global_data_account_pubkey, isSigner: false, isWritable: true },
+        // the slothashes sysvar
```

Comment:
the important information to be conveyed in this comment is not so much what account this is, but rather the purpose it serves for this transaction.
```suggestion
        // program will pull a recent hash from slothashes sysvar if a new valid blockhash is needed.  
```

Reply:
similar comment for `global_data_account_pubkey` above as well as the accounts in `dailyDistributionEvent` (which I failed to note/notice during the earlier PR)